deploy/kubernetes/rbac.yaml

```diff
@@ -28,7 +28,7 @@ rules:
   #   verbs: ["get", "list"]
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "create", "delete"]
+    verbs: ["get", "list", "watch", "update", "create", "delete"]
```

Comment:
@deepakkinni ooc, how did this work in previous version without `update` rbac? is it something came in as requirement in between `alpha` and `beta`?

Reply:
Sorry, this is not going in yet, will update this when we plan to move to beta in 1.25

Reply:
That doesn't look encouraging. It would be good to give this feature some soak time in a Prow job to see whether it works reliably *before* enabling it by default.